contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -63,6 +63,20 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
     event NodeOperatorPenalized(address indexed recipientAddress, uint256 sharesPenalizedAmount);
     event NodeOperatorPenaltyCleared(uint256 indexed nodeOperatorId);
 
+    event ValidatorExitStatusUpdated(
+        uint256 indexed nodeOperatorId,
+        bytes publicKey,
+        uint256 eligibleToExitInSec,
+        uint256 proofSlotTimestamp
+    );
+    event ValidatorExitTriggered(
```

Comment:
Maybe remove this event?

Reply:
We discussed it in private and decided to keep